apps/journeys-admin/src/components/Editor/VideoBlockEditor/VideoBlockEditor.tsx

```diff
@@ -49,7 +70,7 @@ export function VideoBlockEditor({
               ? 'Select Video File'
               : selectedBlock.video.title[0].value
           }
-          caption={selectedBlock?.video?.variant?.hls ?? undefined}
+          caption={language ?? undefined}
```

Comment:
We don't need this anymore since the language state can be undefined right?

```suggestion
          caption={language}
```

Reply:
removed